joomla/joomla-cms
# Create a users group through webservices API - fix for Issue #41113

Pull Request for Issue #41113.

### Summary of Changes

delete required="true" from id field


### Testing Instructions

See 
https://github.com/joomla/joomla-cms/issues/41113

do a webservices api call

curl --location '{{base_url}}api/index.php/v1/users/groups' --header 'Content-Type: application/json' --data '{"title": "Group Title", "parent_id": 1}'

### Actual result BEFORE applying this Pull Request

Response Code: 400 Bad Request - { "errors": [ { "title": "Field required: id" } ] }

### Expected result AFTER applying this Pull Request

Works, created group is returned


### Link to documentations
Please select:
